Add basic routes and dummy components

lotrien opened the pull request.

ikalnitsky commented:
I need your review, guys! /cc @olha-kurkaiedova @malor

malor commented on project.clj:
```diff
@@ -0,0 +1,49 @@
+     :distribution :repo}
+
+  :dependencies
+    [[org.clojure/clojure "1.7.0"]
```
any particular reason not to use the latest versions here? 1.8.0 and 1.9.14 respectively

ikalnitsky replied:
Ehm.. Nope, I just copy pasted it from some template. :) I'll fix it in my next patchset.

malor commented:
Gave it a try, it's a pure magic, but it works :P

lotrien committed:
Teach ESLint about Link component of react-router

Apparently, AirBnB ESLint rules complain about using "to" in Link
component of react-router-dom. Since we are going to use this component
we need to allow this special case to pass the check.

lotrien committed:
Add basic routes and dummy components

ikalnytskyi reviewed (approved):
LGTM. 👍

malor reviewed (approved):
👍 

Thanks, @olha-kurkaiedova ! And thanks @ikalnytskyi for providing insights on how to give it a try locally! ;)
